hseling-web-cat-and-kittens/hseling_web_cat_and_kittens/main.py

```diff
@@ -16,6 +43,22 @@
 
 print(app.config)
 
+mysql = MySQL(app)
+squlitedb = SQLAlchemy(app)
+Login_Manager = LoginManager()
+Login_Manager.init_app(app)
+Login_Manager.login_view = 'login'
+
+app.config['SECRET_KEY'] = 'dmvndshjvbsdvjhvjvgfuy'
```

Comment:
It's not considered the best practice to have keys, tokens or passwords committed into Git.

Instead, try to use data from environment, something like this:

```python
app.config['SECRET_KEY'] = os.environ.get('SECRET_KEY', 'default_secret_key_value')
```

Reply:
change as suggested